### PR TITLE
Raw data summary

### PR DESCRIPTION
**Pull request by robinschmid**

### CODE REVIEW

File: src/main/java/io/github/mzmine/modules/visualization/raw_data_summary/RawDataSummaryPane.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    setCenter(tabPane);
+
+  }
+
+  private BorderPane createScanHistoParameters(final String title, final RawDataFile[] dataFiles,
+      final ScanDataType scanDataType, final ScanSelection scanSelection, final Boolean useMzRange,
+      final Range<Double> mzRange, final Boolean useHeightRange, final Range<Double> heightRange,
+      final boolean useMassDetect, MassDefectFilter massDefectFilter,
+      final ScanHistogramType histoType, final double binWidth) {
+    ScanHistogramParameters mzparams = new ScanHistogramParameters();
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/visualization/raw_data_summary/RawDataSummaryPane.java:119]: .cloneParameterSet or you set the ones from the config

**Review by SteffenHeu — COMMENTED** [review]
nice, just some minor things

File: src/main/java/io/github/mzmine/gui/chartbasics/HistogramChartFactory.java [hunk 2]
```diff
@@ -636,6 +636,10 @@ public static double getMax(double[] data) {
   }
 
   public static Range getBounds(double[] data) {
+    if (data == null || data.length == 0) {
+      return new Range(-1, -1 + Double.MIN_VALUE);
+    }
```
[the remark this answers is not in the document]
Comment (robinschmid) [reply src/main/java/io/github/mzmine/gui/chartbasics/HistogramChartFactory.java:641]: done

File: src/main/java/io/github/mzmine/modules/visualization/raw_data_summary/RawDataSummaryPane.java [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    var mzRange = parameters.getEmbeddedParameterValueIfSelectedOrElse(
+        RawDataSummaryParameters.mzRange, Range.all());
+    var useHeightRange = parameters.getValue(RawDataSummaryParameters.heightRange);
+    var heightRange = parameters.getEmbeddedParameterValueIfSelectedOrElse(
+        RawDataSummaryParameters.heightRange, Range.all());
+    var massDefectFilter = new MassDefectFilter(0.25, 0.4);
+    var upperMzCutoffForMassDefect = 200d;
+
+    var formats = MZmineCore.getConfiguration().getGuiFormats();
+
```
[the remark this answers is not in the document]
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/visualization/raw_data_summary/RawDataSummaryPane.java:71]: done

File: src/main/java/io/github/mzmine/modules/visualization/raw_data_summary/RawDataSummaryPane.java (hunk 1 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/visualization/raw_data_summary/RawDataSummaryPane.java:119]: done

**Pull request merged**